Flake in TestWebhookGitHubPushWithImageStream test

https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin/4235/console
Either a race in the test or in the actual API
```
--- FAIL: TestWebhookGitHubPushWithImageStream-2 (6.82s)
	webhookgithub_test.go:278: Wrong response code, expecting 200, got %!s(int=500): {
		  "kind": "Status",
		  "apiVersion": "v1",
		  "metadata": {},
		  "status": "Failure",
		  "message": "Internal error occurred: could not generate a build: buildconfig \"pushbuild\" cannot be updated: the object has been modified; please apply your changes to the latest version and try again",
		  "reason": "InternalError",
		  "details": {
		    "causes": [
		      {
		        "message": "could not generate a build: buildconfig \"pushbuild\" cannot be updated: the object has been modified; please apply your changes to the latest version and try again"
		      }
		    ]
		  },
		  "code": 500
		}!
FAIL
```

--- a/test/integration/webhookgithub_test.go
+++ b/test/integration/webhookgithub_test.go
@@ -292,10 +292,6 @@ func mockBuildConfigImageParms(imageName, imageStream, imageTag string) *buildap
 						Secret: "secret101",
 					},
 				},
-				{
-					Type:        buildapi.ImageChangeBuildTriggerType,
-					ImageChange: &buildapi.ImageChangeTrigger{},
-				},
 			},
 			BuildSpec: buildapi.BuildSpec{
 				Source: buildapi.BuildSource{
@@ -337,10 +333,6 @@ func mockBuildConfigImageStreamParms(imageName, imageStream, imageTag string) *b
 					GitHubWebHook: &buildapi.WebHookTrigger{
 						Secret: "secret101",
 					},
-				},
-				{
-					Type:        buildapi.ImageChangeBuildTriggerType,
-					ImageChange: &buildapi.ImageChangeTrigger{},
 				},
 			},
 			BuildSpec: buildapi.BuildSpec{

